chromium/background.js

```diff
@@ -118,32 +87,14 @@ chrome.webNavigation.onCompleted.addListener(function() {
   updateState();
 });
 
-/**
-* Load stored user rules
- **/
-var getStoredUserRules = function() {
-  var oldUserRuleString = ls.getItem(USER_RULE_KEY);
-  var oldUserRules = [];
-  if (oldUserRuleString) {
-    oldUserRules = JSON.parse(oldUserRuleString);
-  }
-  return oldUserRules;
-};
-var wr = chrome.webRequest;
-
-/**
- * Load all stored user rules
- */
-var loadStoredUserRules = function() {
-  var rules = getStoredUserRules();
-  var i;
-  for (let rule of rules) {
-    all_rules.addUserRule(rule);
-  }
-  util.log(util.INFO, 'loaded ' + i + ' stored user rules');
-};
-
-loadStoredUserRules();
+// Records which tabId's are active in the HTTPS Switch Planner (see
+// devtools-panel.js).
+var switchPlannerEnabledFor = {};
```

Comment:
It looks like this `switchPlanner` stuff got moved down, but is unchanged?

Reply:
That's right.  I wanted to move it closer to where it is actually used, for clarity.